confirmevent/index.js

```diff
@@ -1,16 +1,50 @@
+const sql = require("mssql");
+
+const sqlConfig = {
+  user: process.env.dbuser,
+  password: process.env.dbpassword,
+  server: process.env.dbserver,
+  database: process.env.database,
+};
+
+async function writeConfirm(eventId, userToken) {
+  try {
+    let currentTimestamp = Math.round(new Date().getTime() / 1000);
```

Comment:
So to be clear, this looks like a `seconds` timestamp, correct? On second thought, maybe millisecond precision is worthwhile?

Reply:
Not likely, but two submissions could happen in the same second. Also, it's nicer dealing with javascript `Date`s in millis.